packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -30,25 +31,42 @@ const DEFAULT_CLI_CONFIG_FILENAME = new URL('client.config.json', DEFAULT_CONFIG
 const LEGACY_CLI_CONFIG_FILENAME = new URL('config.json', DEFAULT_CONFIG_PATH) // todo(1615): Remove this backwards compatibility support
 const DEFAULT_INDEXING_DB_FILENAME = new URL('./indexing.sqlite', DEFAULT_CONFIG_PATH)
 
-const DEFAULT_DAEMON_CONFIG = DaemonConfig.fromObject({
-  anchor: {},
-  'http-api': { 'cors-allowed-origins': [new RegExp('.*')], 'admin-dids': [] },
-  ipfs: { mode: IpfsMode.BUNDLED },
-  logger: { 'log-level': LogLevel.important, 'log-to-files': false },
-  metrics: {
-    'metrics-exporter-enabled': false,
-  },
-  network: { name: Networks.TESTNET_CLAY },
-  node: {},
-  'state-store': {
-    mode: StateStoreMode.FS,
-    'local-directory': DEFAULT_STATE_STORE_DIRECTORY.pathname,
-  },
-  indexing: {
-    db: `sqlite://${DEFAULT_INDEXING_DB_FILENAME.pathname}`,
-    'allow-queries-before-historical-sync': true,
-  },
-})
+/**
+ * Generates a valid Daemon config.
+ *
+ * Most values are set to hardcoded defaults.
+ * The `node.did-seed` is randomly generated.
+ * @returns Daemon config with default values
+ */
+const generateDefaultDaemonConfig = () => {
+  const didSeed = generateSeedUrl()
+  const did = makeNodeDID(parseSeedUrl(didSeed)).id
+
+  return DaemonConfig.fromObject({
+    anchor: {
+      auth: AnchorServiceAuthMethods.DID
+    },
+    'http-api': { 'cors-allowed-origins': [new RegExp('.*')], 'admin-dids': [] },
+    ipfs: { mode: IpfsMode.BUNDLED },
+    logger: { 'log-level': LogLevel.important, 'log-to-files': false },
+    metrics: {
+      'metrics-exporter-enabled': false,
+    },
+    network: { name: Networks.TESTNET_CLAY },
+    node: {
+      'did': did,
```

Comment:
Why is the did itself needed in the config? Makes it harder to manually configure a private key. Could we just output the DID when we start the node?

Reply:
yeah i like this idea